src/sysusers/sysusers.c

```diff
@@ -1550,10 +1553,19 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
 
                                 path_kill_slashes(i->uid_path);
                         } else {
+                                _cleanup_free_ char *uid = NULL, *gid = NULL;
+                                if (split_pair(resolved_id, ":", &uid, &gid) == 0) {
+                                        r = parse_gid(gid, &i->gid);
+                                        if (r < 0) {
+                                                return log_error_errno(r, "Failed to parse GID: '%s': %m", id);
+                                        }
```

Comment:
When only single line in `if()` block, then please do not use `{}`. Of course this is valid. But please read our CODING_STYLE.

Reply:
Sure, fixed! Sorry for this.